service/awsconfig/v4/resource/cloudformation/create.go

```diff
@@ -25,11 +24,11 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 	r.logger.LogCtx(ctx, "debug", "creating AWS cloudformation stack")
 
 	if stackInput.StackName != nil {
-		_, err = r.Clients.CloudFormation.CreateStack(&stackInput)
+		_, err = r.clients.CloudFormation.CreateStack(&stackInput)
```

Comment:
Any "service" implementation should be publicly configurable and deal with the rest privately.